src/Components/DynamicContent/Container.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import AssetManager from './../../Components/DynamicContent/Asset/AssetManager';
+import ConfigManager from './../../Components/DynamicContent/Config/ConfigManager';
+import Search from './../../Components/DynamicContent/Search/Search.js';
+
+const Content = ({view}) => {
+
+    const setContent = (view) => {
+        switch (view) {
+            case "Search":
+                return <Search/>                
+            case "Upload":
+                return <AssetManager/>                
+            case "Config":
+                return<ConfigManager/>                      
+            default:
+                break;
```

Comment:
I usually throw an Error in the default case.
`throw new Error('Unexpected view supplied')`